docs/configuration.md

```diff
@@ -0,0 +1,51 @@
+# Environment Configuration
+
+When developing a Probot App, you will need to have several different fields in a `.env` file which specify environment variables. Here are some common use cases:
+
+```
+WEBHOOK_PROXY_URL='https://smee.io/yourcustomurl'
+# This is used to allow smee to communicate as your Webhook URL
```

Comment:
What's a smee and why does it need to communicate as my webhook url? :trollface: 

Alternate suggestion:

> Allows your local development environment to receive GitHub webhook events. Go to https://smee.io/new to get started.